platform/cluster.go

```diff
@@ -169,6 +169,14 @@ func (bc *BaseCluster) RenderUserData(userdata *conf.UserData, ignitionVars map[
 		conf.CopyKeys(keys)
 	}
 
+	// disable Zincati & Pinger by default
+	if bc.Distribution() == "fcos" {
+		conf.AddFile("/etc/fedora-coreos-pinger/config.d/90-disable-reporting.toml", "root", `[reporting]
+enabled = false`, 0644)
+		conf.AddFile("/etc/zincati/config.d/90-disable-auto-updates.toml", "root", `[updates]
```

Comment:
FYI `update.enabled = true` (as a single line) is also valid in TOML as an equivalent format, if you prefer that syntax.